### PR TITLE
Update Ficha Tab with candidate data

### PR DESCRIPTION
**Pull request by cristianbgp**
Includes #12 

Adds data from candidate
It might still need some tweaks in smaller screens

### CODE REVIEW

File: components/Tabs/Ficha.js [hunk 1]
```diff
@@ -14,59 +23,82 @@ export default function Ficha() {
...
+                <Completed className="ml-3" />
+              ) : (
+                <Incompleted className="ml-3" />
+              )}
             </span>
           </div>
         </div>
         <div className="px-6 pt-4 pb-6">
           <h4 className="text-xs text-neutral-400 font-semibold">
             EDUCACIÓN UNIVERSITARIA
```
Comment (lavaldi) [inline components/Tabs/Ficha.js:44]: I wonder what will be shown if the candidate doesn't have a college education or work experience 🤔 cc @ederrengifo

Comment (cristianbgp) [reply components/Tabs/Ficha.js:44]: Also I just noticed that we have postgrado data that we are not showing 😅

Comment (lavaldi) [reply components/Tabs/Ficha.js:44]: I suppose that postgraduate degrees should be shown in university education as well ☝️

**Review by lavaldi — APPROVED** [review]
I think we can approve this PR and make the necessary corrections in another, regarding what would be shown in the sections that don't have information and show the postgraduate information

**Pull request merged**